docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        return first_letter.substitute(election_types, else_=ibis.null())
</pre><p><code>ibis.NA</code> is also fine here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Done</p>